packages/expo-constants/package.json

```diff
@@ -36,11 +36,11 @@
     "unimodules-constants-interface": "*"
   },
   "dependencies": {
-    "ua-parser-js": "^0.7.19"
+    "ua-parser-js": "^0.7.19",
+    "uuid-js": "^0.7.5"
```

Comment:
We use uuid in doument-picker -> pulling in multiple UUID libraries is going to be more expensive than pulling in just one. I wouldn't over-optimize for this constants package since it's the least useful out of them all.